prov/efa/src/efa_cntr.c

```diff
@@ -44,6 +44,13 @@ static int efa_cntr_wait(struct fid_cntr *cntr_fid, uint64_t threshold, int time
 	int numtry = 5;
 	int tryid = 0;
 	int waitim = 1;
+	struct util_srx_ctx *srx_ctx;
+	struct efa_cntr *efa_cntr;
+
+	efa_cntr = container_of(cntr_fid, struct efa_cntr, util_cntr.cntr_fid);
+	srx_ctx = efa_cntr->util_cntr.domain->srx->ep_fid.fid.context;
```

Comment:
Could this be NULL?

Reply:
Good call, srx is only available for RDM ep type. I will add a check here (and all cntr ops)

Reply:
It seems we never support counter for DGRAM ep type. But I will still add such check for correctness.

Reply:
But I will still add such check for correctness